src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,139 @@
+package duke;
+
+import java.util.Scanner;
+
+import duke.task.Task;
+
+/**
+ * The Ui class deals with interactions with the user.
+ *
+ * @author Edric Yeo
+ */
+public class Ui {
+
+    private Scanner sc;
```

Comment:
Do you think it would be better to create the Scanner object in the readCommand method as it is the only function in which it is used?